webapp/portlet/src/main/webapp/idm-users-management/components/UsersManagementList.vue

```diff
@@ -29,6 +29,21 @@
       :no-results-text="$t('UsersManagement.noResultsFound')"
       :no-data-text="$t('UsersManagement.noData')"
       class="data-table-light-border">
+      <template slot="item.lastConnexion" slot-scope="{ item }">
+        <div v-if="typeof item.lastConnexion == 'number'">
+          <date-format
+            :value="item.lastConnexion"
+            :format="fullDateFormat"
+            class="mr-1" />
+          <date-format
+            :value="item.lastConnexion"
+            :format="dateTimeFormat"
+            class="mr-1" />
```

Comment:
```suggestion
          <date-format
            :value="item.lastConnexion"
            :format="fullDateFormat"
            class="mr-1" />
```
You can combine both by using this object:

```
    fullDateFormat: {
      day: 'numeric',
      month: 'short',
      year: 'numeric',
      hour: '2-digit',
      minute: '2-digit',
    }
```